pandas/tests/series/test_rank.py

```diff
@@ -196,9 +196,6 @@ def test_rank_signature(self):
         pytest.raises(ValueError, s.rank, 'average')
 
     def test_rank_inf(self):
-        pytest.skip('DataFrame.rank does not currently rank '
```

Comment:
can you add some nans in here as well. is this enough of a test?

Reply:
Yes, I can add nans there.